data/mods/Magiclysm/Spells/technomancer.json

```diff
@@ -51,7 +51,7 @@
   {
     "id": "holy_blade",
     "type": "SPELL",
-    "name": "Holy Blade",
+    "name": "Mage Blade",
     "description": "This blade of light will cut through any evil it makes contact with!",
```

Comment:
Is it supposed to still mention evil despite having the name Mage Blade?

Reply:
Perhaps something along the lines of "Blade of hardened mana" would be more apt.